src/game.js

```diff
@@ -1,37 +1,44 @@
-const dbGame = []
 const idHelper = require('./IdHelper.js')
+const GameBd = require('.././database/GameModel.js')
 
 class Game {
-	constructor({cols = 10, rows = 10} = {}){
-		this.cols = cols;
-		this.rows = rows;
-	}
-	static create({cols = 10, rows = 10} = {}) {
-		const game = new Game({cols,rows});
-		game.id = dbGame.length + 1;
-		game.playerId = idHelper();
-		const token = idHelper();
-		game.token = token
-		dbGame.push(game);
-		game.session = `http://localhost:3000/game?token=${token}`;
+  constructor({cols = 10, rows = 10} = {}) {
+    this.cols = cols;
+    this.rows = rows;
+  }
 
-		return Promise.resolve({
-			id : game.id, 
-			session : game.session,
-			playerId : game.playerId
-		})
-	}
+  static create({cols = 10, rows = 10} = {}) {
+    const game = new Game({cols,rows});
+    game.playerId = idHelper();
+    const token = idHelper();
+    game.token = token
+    game.session = `http://localhost:3000/game?token=${token}`;
+    return GameBd.sync()
+      .then(() => GameBd.create({
+        token,
+        playerId1 : game.playerId
+      }))
+  }
 
-	static join(token) {
-		const game = dbGame.find(game => game.token === token);
-		if(game === undefined) {
-			return Promise.reject()		
-		}
-		return Promise.resolve({
-			id : game.id,
-			playerId : idHelper()
-		});
-	}
+  static join(token) {
+    const game = GameBd.findAll(
+    { 
+      where: { 
+        token: token 
+      }
+    })
+    if(game === undefined) {
+      return Promise.reject()
+    }
+    GameBd.update({ 
+        playerId2: idHelper() 
+      }, { 
+        where: { 
+          token: token 
+        }
+      })
+    return Promise.resolve(game);
+  }
```

Comment:
promises are not being used properly here, the should be chained, otherwise you are not getting the right responses consistenly